.github/workflows/pylint.yml

```diff
@@ -37,16 +37,16 @@ jobs:
         run: sudo apt-get install python3-pip
 
       # Install python3-venv
-      - name: install python3-venv
-        run: sudo apt-get install python3-venv
+      # - name: install python3-venv
+      #   run: sudo apt-get install python3-venv
 
-      # Create virtual env
-      - name: create virtual env
-        run: python3 -m venv venv
+      # # Create virtual env
+      # - name: create virtual env
+      #   run: python3 -m venv venv
 
-      # Activate virtual env
-      - name: activate virtual env
-        run: source venv/bin/activate
```

Comment:
Commented out venv for constantly failing to install python3-venv package. Linting passed and I think we can merge this pr now.

Reply:
Do we know why it failed all of a sudden?

Reply:
I don't know for sure, probably because the environment on lint server changed so additional set up is required (e.g. update apt-get).